api/auth.ts

```diff
@@ -97,7 +97,7 @@ type AuthenticateMiddleware = (
  * Authenticate a JWT in the 'Authorization' header of the given type
  */
 const authenticate = function (
-  type: string | null,
+  types: string[] | null,
```

Comment:
This could potentially break other peoples code doing anything with our api.  What is the reason for this change?

Reply:
This isn't called anywhere direclty outside of this class, and i have made all the authentication methods work with this change